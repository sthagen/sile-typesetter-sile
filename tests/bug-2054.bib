@phdthesis{foo,
  author  = {Doe, John and Smith, Jane and Bob and Alice},
  title   = {A Tale of Four Authors},
}

@book{bar,
  author     = {Doe, John and Smith, Jane},
  translator = {Pumpernickel, Peter},
  title      = {A Book},
}
