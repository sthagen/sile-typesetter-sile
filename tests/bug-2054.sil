\begin[papersize=a6]{document}
\nofolios
\neverindent
\language[main=en]
\use[module=packages.bibtex]
\set[parameter=bibtex.style, value=chicago]% LEGACY IMPLEMENTATION
\loadbibliography[file=tests/bug-2054.bib]

\reference{bar}

\reference{foo}

% Should not use Latin localization, should match first use of reference
\reference{bar}

\end{document}
