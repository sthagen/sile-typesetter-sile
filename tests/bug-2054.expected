Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	28.5447
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	39 82 72 w=15.5762	(Doe)
Mx 	30.4581
T	15 w=2.2900	(,)
Mx 	34.9516
T	45 w=3.0518	(J)
Mx 	38.0033
T	17 w=2.2900	(.)
Mx 	42.4969
T	68 81 71 w=15.3076	(and)
Mx 	60.0079
T	54 80 76 87 75 w=24.4629	(Smith)
Mx 	84.4708
T	15 w=2.2900	(,)
Mx 	88.9643
T	45 w=3.0518	(J)
Mx 	92.0161
T	17 w=2.2900	(.)
Mx 	94.3061
T	15 w=2.2900	(,)
Mx 	98.7997
Set font 	Gentium Plus;10;400;Italic;normal;;;LTR
T	36 w=5.6006	(A)
Mx 	106.4034
T	37 82 82 78 w=17.7783	(Book)
Mx 	124.1817
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	17 w=2.2900	(.)
Mx 	128.6752
T	55 85 68 81 86 79 68 87 72 71 w=43.9990	(Translated)
Mx 	174.8777
T	69 92 w=10.0098	(by)
Mx 	187.0910
T	51 72 87 72 85 w=21.8701	(Peter)
Mx 	211.1646
T	51 88 80 83 72 85 81 76 70 78 72 79 w=57.2412	(Pumpernickel)
Mx 	268.4058
T	17 w=2.2900	(.)
Mx 	14.8819
My 	40.5447
T	39 82 72 w=15.5762	(Doe)
Mx 	32.6737
T	72 87 w=8.0615	(et)
Mx 	42.9508
T	68 79 w=7.2998	(al)
Mx 	50.2506
T	17 w=2.2900	(.)
Mx 	52.5406
T	15 w=2.2900	(,)
Mx 	57.0463
T	180 w=4.3604	(“)
Mx 	61.4066
T	36 w=5.9521	(A)
Mx 	69.5744
T	55 68 79 72 w=17.4268	(Tale)
Mx 	89.2167
T	82 73 w=8.1396	(of)
Mx 	99.5720
T	41 82 88 85 w=19.0674	(Four)
Mx 	120.8550
T	36 88 87 75 82 85 86 w=33.0615	(Authors)
Mx 	153.9165
T	17 w=2.2900	(.)
Mx 	156.2065
T	181 w=4.3604	(”)
Mx 	14.8819
My 	52.5447
T	39 82 72 w=15.5762	(Doe)
Mx 	30.4581
T	15 w=2.2900	(,)
Mx 	34.9516
T	45 w=3.0518	(J)
Mx 	38.0033
T	17 w=2.2900	(.)
Mx 	42.4969
T	68 81 71 w=15.3076	(and)
Mx 	60.0079
T	54 80 76 87 75 w=24.4629	(Smith)
Mx 	84.4708
T	15 w=2.2900	(,)
Mx 	88.9643
T	45 w=3.0518	(J)
Mx 	92.0161
T	17 w=2.2900	(.)
Mx 	94.3061
T	15 w=2.2900	(,)
Mx 	98.7997
Set font 	Gentium Plus;10;400;Italic;normal;;;LTR
T	36 w=5.6006	(A)
Mx 	106.4034
T	37 82 82 78 w=17.7783	(Book)
Mx 	124.1817
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	17 w=2.2900	(.)
Mx 	128.6752
T	55 85 68 81 86 79 68 87 72 71 w=43.9990	(Translated)
Mx 	174.8777
T	69 92 w=10.0098	(by)
Mx 	187.0910
T	51 72 87 72 85 w=21.8701	(Peter)
Mx 	211.1646
T	51 88 80 83 72 85 81 76 70 78 72 79 w=57.2412	(Pumpernickel)
Mx 	268.4058
T	17 w=2.2900	(.)
End page
Finish
